src/pages/ScreenRouting/mediaScreens/MediaEpisode/MediaEpisode.tsx

```diff
@@ -0,0 +1,32 @@
+import { Navigate } from 'react-router';
+
+import type { ScreenComponent } from '#types/screens';
+import type { PlaylistItem } from '#types/playlist';
+import Loading from '#src/pages/Loading/Loading';
+import { mediaURL } from '#src/utils/formatting';
+import useQueryParam from '#src/hooks/useQueryParam';
+import { useSeriesLookup } from '#src/hooks/series/useSeriesLookup';
+import useMedia from '#src/hooks/useMedia';
+
+/**
+ * This media series episode screen is used to redirect an episode item to the series page.
+ */
+const MediaEpisode: ScreenComponent<PlaylistItem> = ({ data: media, isLoading: isMediaLoading }) => {
+  const mediaId = media.mediaid;
+
+  const play = useQueryParam('play') === '1';
+  const feedId = useQueryParam('r');
+
+  const { data: episodeInSeries, isLoading: isSeriesIdLoading } = useSeriesLookup(media.mediaid);
```

Comment:
For legacy series, we won't get a result for `episodesInSeries`. I think it's fine to let this be the trigger for the fallback and to do the legacy series id look and redirecting in this component.